spec/harness_spec.rb

```diff
@@ -0,0 +1,167 @@
+require '../futbol/lib/stat_tracker'
```

Comment:
Thanks for including the harness, this will make it to where we can keep track of whether our methods are working to the instructors' expectations